cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -978,6 +1016,38 @@ private void configureQuotasPluginStrimzi(String clusterName, QuotasPluginStrimz
         writer.println(String.format("client.quota.callback.static.excluded.principal.name.list=%s", String.join(";", excludedPrincipals)));
     }
 
+    /**
+     * Append list configuration values or create a new list configuration if missing.
+     * A list configuration can contain a comma separated list of values.
+     * Duplicated values are removed.
+     *
+     * @param kafkaConfig Kafka configuration.
+     * @param key List configuration key.
+     * @param values List configuration values.
+     */
+    static void createOrAddListConfig(AbstractConfiguration kafkaConfig, String key, String values) {
+        if (kafkaConfig == null) throw new IllegalArgumentException("Configuration is required");
+        if (key == null || key.isBlank()) throw new IllegalArgumentException("Configuration key is required");
+        if (values == null || values.isBlank()) throw new IllegalArgumentException("Configuration values are required");
```

Comment:
I think if you properly format it with `{` / `}`, it would be more readable and clearer what exactly are you doing there. I do not think we use this form in many parts of the code base, if somewhere at all.